src/Code/Workspace/WorkspaceItem.elm

```diff
@@ -855,6 +861,35 @@ decodeTypes ref =
     Decode.keyValuePairs decodeTypeDetails |> Decode.map buildTypes
 
 
+decodeTypesWithRef : Decode.Decoder (List ( Reference, TypeDetailWithDoc ))
+decodeTypesWithRef =
+    let
+        makeType ( hash_, d ) =
+            let
+                ref =
+                    Reference.fromFQN Reference.TypeReference d.name
```

Comment:
In this part, `ref` is constructed based on `bestTypeName` (`d.name`) of the API response. 
I'm not sure if this works in all cases. Do you think this is fine?

Reply:
This didn't work with actual projects. I'll check and fix it soon